tests/gold_tests/pluginTest/tsapi/tsapi.test.py

```diff
@@ -26,6 +26,13 @@
 )
 Test.ContinueOnFail = True
 
+plugin_name = "test_tsapi"
+log_file_name = "log.txt"
+
+
+# The test plugin will output test logging to this file.
+Test.Env["OUTPUT_FILE"] = Test.RunDirectory + "/" + log_file_name
```

Comment:
A small thing, but better to use `os.path.join` here as is done elsewhere in the file.